app/src/utils/FakeMetadataMap.ts

```diff
@@ -28,7 +28,7 @@ const fakeMetadata: DatasetMetadata[] = [
       },
       {
         data_type: "string",
-        name: "race",
+        name: "hispanic_or_latino_and_race",
```

Comment:
maybe rename to ethnicity_and_race ? at first glance I was confused thinking this variable referred to hispanic/latino ethnicity only

Reply:
Can I do this as a followup? I've littered this constant throughout the code and I'm worried about breaking things. I want to refactor commonly used column names to not be hard-coded anyway, so I can change this as part of that.